schema/cpacs_schema.xsd

```diff
@@ -35975,6 +35975,7 @@ The fuel tank volume type should also be used for the wing fuel tank</xsd:docume
                     <xsd:element name="transformation" minOccurs="0" type="transformationType"/>
                     <xsd:element name="positionings" minOccurs="0" type="positioningsType"/>
                     <xsd:element name="segments" type="fuselageSegmentsType"/>
+                    <xsd:element name="sections" type="fuselageSectionsType"/>
```

Comment:
Ups, I updated the official scheme accordingly https://github.com/DLR-SL/CPACS/commit/f5c31da64fea509d1c3eb8d7d10928e72f6fb6fa

Reply:
Thanks @MarAlder. I suppose this PR can be closed then?